sechub-web-ui-solution/helm/sechub-web-ui/values.yaml

```
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: MIT
+
+# This is a sample values file containing the defaults.
+
+name: web-ui
+
+image: "ghcr.io/mercedes-benz/sechub/sechub-web-ui:latest"
+
+# Optional: If your image is in a private registry, you can specify pull secrets defined in k8s
+#   Docs: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+#   Example:
+# imagePullSecrets: |-
+#   - name: sechub-pull-secret
+#   - name: sps2
+
+# Number of instances to spin up
+replicaCount: 1
+
+resources:
+  requests:
+    # Initial container memory size
+    memory: 256Mi
+  limits:
+    # Maximum container memory size
+    memory: 1Gi
+
+web_ui:
+  ssl:
+    keystoreAlias: "undefined"
+    # Alias in .p12 keystore.
+    # - On `undefined`, a self-signed certificate will be used.
+    # - otherwise, k8s secret `secret-webui-ssl` must be defined containing
+    #   - `keystore_file` (containing the ssl certificate chain)
+    #     Inside the .p12 keystore, an alias with this name is expected
+    #     pointing to the ssl certificate to use
+    #   - `keystore_password`
+
+# deploymentComment (optional):
+#  When setting to a different value than before, it forces k8s to spin up a new container.
+#  This way, you can force deployments e.g. when only secrets have changed.
+deploymentComment: "my deployment comment"
+
+service:
+  loadbalancer:
+    ip: ""
+    port:
+      name: web-ui-https
+      number: 8443
```

8443 is already occupied by the SecHub server. We need a different default port
```suggestion
      number: 4443
```